### ISSUE TITLE
Submit validation errors in FieldArray are initialised with empty

### ISSUE THREAD

**Issue by a-xin** [erikras/redux-form#2667]
### Are you submitting a **bug report** or a **feature request**?
Bug

### What is the current behavior?
- Create a `FieldArray` with the possibility to add fields (like in the FieldArray example)
- Add a field
- Use Submit Validation and throw an error for the submitted field
  ```
  submitErrors: {
    subfields: ['invalid input']
  }
  ```
- Add an additional field to the beginning of the `FieldArray`
=> The additional field receives a submit error `empty` (ie. `{ }`)
  ```
  submitErrors: {
    subfields: [{ }, 'invalid input']
  }
  ```

### What is the expected behavior?
The additional field receives a submit error `undefined`
```
submitErrors: {
  subfields: [undefined, 'invalid input']
}
```

**Comment by gustavohenke** [comment]
Fixed by #2668

**Comment by erikras** [comment]
Fix published in [`v6.6.0`](https://github.com/erikras/redux-form/releases/tag/v6.6.0).
